aioftp/client.py

```diff
@@ -7,6 +7,8 @@
 import pathlib
 import re
 from functools import partial
+from typing import Optional, Union, Type, Tuple, Dict, List
```

Comment:
We can target python 3.11, I think for better typing style like `x: int | float | None = None`

Reply:
`x: int | float | None = None`  was first added to Python3.10 ,https://docs.python.org/3/whatsnew/3.10.html 
 [PEP 604](https://peps.python.org/pep-0604/), Allow writing union types as X | Y
So,  we'll target 3.10+ ?

Reply:
https://pythonspeed.com/articles/stop-using-python-3.7/ says:
As of June 2023, [about 30% of packages downloaded from PyPI were for Python 3.7](https://pypistats.org/packages/__all__).

Reply:
It is okay for now. Python release cycle has changed, and 3.7 has EOL a week ago (https://endoflife.date/python). Also, there are no current plans to add much features to project. So, I see no reason to not to target 3.10 or 3.11.